extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -235,7 +235,7 @@ protected static function onLoggableUserIPData( IContextSource $context, array $
 			'cuc_title'      => $data['title'], // may be ''
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app